Flutter/lib/main.dart

```diff
@@ -47,14 +48,8 @@ class _MyHomePageState extends State<MyHomePage> {
   int _counter = 0;
 
   void _incrementCounter() {
-    setState(() {
```

Comment:
Bro envez de hacer esto why don't you make the login in page the main view for now?

Reply:
pensaba que era mejor dejarlo asi hasta que terminemos el main view. Tenia entendido que el main view iba a decidir si mandar el usuario al login page o a registration.

Reply:
Si yo concuerdo con nico vamos a dejarlo todo como esta until we figure our the correct way for the main page handler. asi no hay que mover codigo a cada momento